### DISCUSSION TITLE
[Feature] Word by word completion

### DISCUSSION THREAD

**Discussion by pidgeon777** [community/community#16299] [Copilot Conversations]
First of all, congratulations on the Copilot project.

I'm using the `copilot.vim` plugin for Neovim: what it is really missing, is a hotkey to also complete the suggestion word by word.

It would be a great enhancement. Do you think something like that could be implemented?

**Comment by cmp-nct** [comment]
I added it as a plugin: https://github.com/cmp-nct/Stewardess

**Comment by pidgeon777** [comment]
@cmp-nct 

Will it work also for Neovim (Vim)?

**Reply by cmp-nct** [reply]
Each IDE has a completely different Plugin written by different people I guess.
The one in IntelliJ is extremely closed, it hides everything from the IDE/user and tries to protect the answers with private package walls.
That's what it was very difficult for me to find an actual path to tweak Copilot in IntelliJ.
The plugin I wrote will only work with IntelliJ

I've good news about Neovim though!  During my researches I stumbled on code snippets that offer a simple word-by-word completion in that IDE. I am sure you can find it if you google for a while. It was just 10-15 lines long.
From what I saw the Neovim plugin is not (or can not) hide what it is doing, so it's possible to just access the completion directly.

**Comment by pidgeon777** [comment]
Maybe this could be an interesting topic for @tpope?
